pkg/cri/resource-manager/cache/utils.go

```diff
@@ -127,10 +108,10 @@ func MilliCPUToQuota(milliCPU int64) (int64, int64) {
 		return 0, 0
 	}
 
-	period := int64(QuotaPeriod)
-	quota := (milliCPU * period) / milliCPUToCPU
-	if quota < minQuotaPeriod {
-		quota = minQuotaPeriod
+	period := int64(kubecm.QuotaPeriod)
```

Comment:
This may be a silly question, but is this a constant or does it change with Kubelet's `CPUCFSQuotaPeriod` setting?

Reply:
Not silly, it's a very valid question. The answer is that this is a constant but it is only used if the CPUCFSQuotaPeriod feature gate is disabled (the default). Otherwise the period is taken from the configured CPUCFSQuotaPeriod value.

This code assumed we're running with the default settings. I haven't taken a look at whether and how to do the reverse calculation in the other case. I can give it a try.